### PR TITLE
chore: update validate-release guide

### PR DESCRIPTION
**Pull request by sheli00**
update validate-release guide, delete duplicate content, add Java11 hint.

### CODE REVIEW

File: content/cn/docs/contribution-guidelines/validate-release.md
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/validate-release.md:1]: thanks, also fix the other 2 points & then sync EN doc together~

Comment (sheli00) [reply content/cn/docs/contribution-guidelines/validate-release.md:1]: fix

File: content/cn/docs/contribution-guidelines/validate-release.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ weight: 3
 ---
 
 > Note: 这篇文档会持续更新。
+> 建议使用 Java11 验证，新的版本不再支持 Java8
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/validate-release.md:8]:
```suggestion
> 建议使用 Java11 验证测试，从 1.5.0 版本开始(除 client 外) 不再支持 Java8
```

File: content/en/docs/contribution-guidelines/validate-release.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ weight: 3
 ---
 
 > Note: this doc will be updated continuously.
+> It is recommended to use Java11 verification, the new version no longer supports Java8
```
Comment (imbajin) [inline content/en/docs/contribution-guidelines/validate-release.md:8]:
```suggestion
> It is recommended to use Java11 in verification, we will drop Java8 support from version 1.5.0
```

**Review by imbajin — APPROVED** [review]

**Pull request merged**